frontend/src/features/admin-form/common/queries.ts

```diff
@@ -51,16 +52,18 @@ export const useAdminForm = (
 
 export const useFreeSmsQuota = () => {
   const { formId } = useParams()
-  if (!formId) throw new Error('No formId provided')
+  if (!formId) throw new Error('No formId provided to useFreeSmsQuota')
 
   return useQuery(adminFormKeys.freeSmsCount(formId), () =>
     getFreeSmsQuota(formId),
   )
 }
 
 export const useAdminFormCollaborators = () => {
-  const { formId } = useParams()
-  if (!formId) throw new Error('No formId provided')
+  const { formId } = useCollaboratorWizard()
+  if (!formId)
+    throw new Error('No formId provided to useAdminFormCollaborators')
```

Comment:
same comment as above, no need to check for undefined formId if it was required in the first place